static/js/breakpoint-handler.js

```diff
@@ -22,18 +24,18 @@ function getViewportDimensions() {
  *  enter: callback when breakpoint region entered,
  *  leave: callback when breakpoint region exited
  * }
- * 
+ *
  */
 
-BreakpointHandler = function (opts) {
-    opts || (opts = {});
+function BreakpointHandler(opts) {
```

Comment:
Was this being called before it was defined? Curious why the switch to a declaration instead of expression